frontend/src/api/api.ts

```diff
@@ -499,6 +499,44 @@ export async function createCheckoutSession({ userId, priceId, successUrl, cance
     return session;
 }
 
+export async function getCustomerId({ subscriptionId }: { subscriptionId: any }): Promise<any> {
```

Comment:
@egdagger  Use proper types instead of any to leverage TypeScript effectively. is the subcriptionId string?

```node
export async function getCustomerId({ subscriptionId }: { subscriptionId: string }): Promise<string> {

```